migrations/059_add_linkshare_table.php

```diff
@@ -11,12 +11,12 @@ public function up()
         $db = DBManager::get();
 
         $db->exec("
-            CREATE TABLE `oc_video_shares` (
-                `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
-                `token` varchar(32) NOT NULL,
-                `video_id` int NOT NULL,
-                `password` varchar(255) NULL,
-                FOREIGN KEY (`video_id`) REFERENCES `oc_video` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+            CREATE TABLE IF NOT EXISTS `oc_video_shares` (
```

Comment:
Please move this to a new migration and do an alter table, because this migration already ran at the customers production system!